embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/opentelemetry/EmbSpan.kt

```diff
@@ -63,13 +70,18 @@ internal class EmbSpan(
      * the underlying span is set to [StatusCode.OK] automatically if this is called before [setStatus] is called.
      */
     override fun end(timestamp: Long, unit: TimeUnit) {
-        val endTimeMs = unit.toMillis(timestamp)
-        when (spanStatus) {
-            StatusCode.ERROR -> {
-                embraceSpan.stop(errorCode = ErrorCode.FAILURE, endTimeMs = endTimeMs)
-            }
-            else -> {
-                embraceSpan.stop(endTimeMs = endTimeMs)
+        if (isRecording) {
```

Comment:
Note - the isRecording check isn't strictly necessary but I don't want this wrapper class to be inconsistent with the wrapped `SdkSpan` so I made sure it won't be.